EIPS/eip-2537.md

```diff
@@ -343,17 +337,15 @@ Required properties for pairing operation:
 - Degeneracy `e(P, 0*Q) = e(0*P, Q) = 1` 
 - Bilinearity `e(a*P, b*Q) = e(a*b*P, Q) = e(P, a*b*Q)` (internal test, not visible through ABI)
 
-Test vector for all operations are expanded in this `csv` files in [repo](https://github.com/matter-labs/eip1962/tree/master/src/test/test_vectors/eip2537) as hex encodings of the inputs and outputs to the corresponding precompiles.
-
 ### Benchmarking test cases
 
 A set of test vectors for quick benchmarking on new implementations is located in a separate [file](../assets/eip-2537/bench_vectors.md)
 
 ## Reference Implementation
 
-There are two fully spec compatible implementations:
-- One in Rust language that is based on the EIP1962 [code](https://github.com/matter-labs/eip1962). There exists an integration with OpenEthereum for this library
-- One implemented specifically for Geth as a part of the current [codebase](https://github.com/ethereum/go-ethereum/tree/master/crypto/bls12381)
+There are two fully spec compatible implementations on the day of writing:
+- One in Rust language that is based on the EIP1962 code and integrated with OpenEthereum for this library
+- One implemented specifically for Geth as a part of the current codebase
```

Comment:
While the removal of the links makes this technically valid, it suffers the same problem still which is that it is referencing stuff that may not be meaningful to future readers.  For example, a similar reference to an implementation in `Parity` in an earlier EIP would be fairly worthless today since `Parity` as a product no longer exists.

EIPs are technical documents, and not an appropriate place to track lists of things (like lists of implementations).  I recommend just removing this section entirely since it is optional.